project/standard/templates/web/pom.xml

```diff
@@ -15,14 +15,25 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <tomcat.version>8.5.69</tomcat.version>
     <tomcat.port>8080</tomcat.port>
+    <mapstore-services.version>1.8-SNAPSHOT</mapstore-services.version>
+    <geostore-webapp.version>2.1-SNAPSHOT</geostore-webapp.version>
```

Comment:
I think MS master should goes to 2.2-SNAPSHOT now for this pom file @offtherailz.